### ISSUE TITLE
Duplicate IDs on accordion content

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14501]
## Describe the defect
On each accordion, there is a wrapping `<div>` with a `id="content'`. However, when there are multiple accordions on the page, this becomes a duplicate ID error. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/3f4c6fef-a4a7-4155-9af0-9f7e61f25e96)

There is also an issue because for each accordion, the title button has an `aria-controls` attribute that points to the accordion content `<div>` using the ID. But if each `<div>` has the same ID this could become confusing for assistive technology users.

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/dd602530-76b7-4c77-91a0-6125ff87ce42)

## Example page where this is happening:
https://www.va.gov/boston-health-care/work-with-us/internships-and-fellowships/trainee-programs/

## AC / Expected behavior

- [ ] The ID for each wrapping `<div>` for the accordion content should be unique
- [ ] The `aria-controls` attribute of the accordion button should be the unique ID of the appropriate accordion content
- [ ] Accessibly review

# Additional Info
This issue is currently being flagged in the new AMP automated scans
